tools/plugins/com.liferay.ide.ui/src/com/liferay/ide/ui/LiferayUpgradePerspectiveFactory.java

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+package com.liferay.ide.ui;
+
+import org.eclipse.jdt.ui.JavaUI;
+import org.eclipse.ui.IFolderLayout;
+import org.eclipse.ui.IPageLayout;
+import org.eclipse.ui.progress.IProgressConstants;
+
+/**
+ * @author Lovett Li
+ */
+public class LiferayUpgradePerspectiveFactory extends AbstractPerspectiveFactory
+{
+    public static final String ID = "com.liferay.ide.eclipse.ui.perspective.liferaycodeupgrade";
+
+    @Override
+    public void createInitialLayout( IPageLayout layout )
+    {
+        createLayout(layout);
+        addShortcuts(layout);
+    }
+
+    private void createLayout( IPageLayout layout )
+    {
+        String editorArea = layout.getEditorArea();
+        layout.setEditorAreaVisible( false );
```

Comment:
We should remain this line because of it can help us hidden extra editor area as default. It will not affect open other views.